### ISSUE TITLE
Export file is not encrypted

### ISSUE THREAD

**Issue by iamrearwindow** [beemdevelopment/Aegis#449]
###### Info

* __Version__: 1.1.4
* __Source__: Google Play
* __Vault encrypted:__ Yes (with password unlock)
* __Device:__ Samsung Galaxy S7, SM-G930F
* __Android version and ROM:__ Androd 8.0.0, Samsung ROM G930FXXS8ETC1

###### Steps to reproduce

Screen security enabled.
Auto lock enabled.

In settings hit export, keep the vault encrypted ticked, hit ok.
Save file to default location, internal storage.
See vault has been exported pop-up.

Connect phone to PC via USB.
Copy aegis_export.json file to PC drive.
The file is plain text and I can view it with notepad++.

###### What do you expect to happen?
I expected the export file to be encrypted and unreadable.

###### What happens instead?
The exported file was in plain text.

**Comment by alexbakker** [comment]
The fact that you can open the vault file in Notepad does not mean that your vault is not encrypted. In layman's terms, the ``header`` section of encrypted vaults is plain text and contains some information about how the vault is encrypted (don't worry, it's useless without your password) . The``db`` section of encrypted vaults is not readable You can learn more about the format here: https://github.com/beemdevelopment/Aegis/blob/master/docs/vault.md.

Please take a look at the example vault files here: https://github.com/beemdevelopment/Aegis/tree/master/testdata. Which one of the two looks similar to your vault file?

**Comment by iamrearwindow** [comment]
Hi Alexander,
      
      Thanks for the explanation.
      I kind if expected that I misunderstood something.
      All is good then.
      
      I like the app I must say.
      Very simple and I really want to be able to save my authentication
      keys somewhere other than the cloud.
      
      Regards, Ian
      
      On 31/05/2020 20:32, Alexander Bakker wrote:
    
    
      
      The fact that you can open the vault file in Notepad does not
        mean that your vault is not encrypted. In layman's terms, the header
        section of encrypted vaults is plain text and contains some
        information about how the vault is encrypted (don't worry, it's
        useless without your password) . Thedb section of
        encrypted vaults is not readable You can learn more about the
        format here: https://github.com/beemdevelopment/Aegis/blob/master/docs/vault.md.
      Please take a look at the example vault files here: https://github.com/beemdevelopment/Aegis/tree/master/testdata.
        Which one of the two looks similar to your vault file?
      —
        You are receiving this because you authored the thread.
        Reply to this email directly, view it on GitHub, or unsubscribe.
      [
{
"@context": "http://schema.org",
"@type": "EmailMessage",
"potentialAction": {
"@type": "ViewAction",
"target": "https://github.com/beemdevelopment/Aegis/issues/449#issuecomment-636517516",
"url": "https://github.com/beemdevelopment/Aegis/issues/449#issuecomment-636517516",
"name": "View Issue"
},
"description": "View this Issue on GitHub",
"publisher": {
"@type": "Organization",
"name": "GitHub",
"url": "https://github.com"
}
}
]
